Dockerfile

```diff
@@ -10,3 +10,5 @@ COPY . /go/src/app
 WORKDIR /go/src/app
 RUN go-wrapper download
 RUN go-wrapper install
+RUN go build
+ENTRYPOINT ["./app"]
```

Comment:
`make containerrun` doc implies that this will start automatically, but that simply wasn't the case.  I was popped into a shell where I had to `go run main.go`